themes/sdss_subtheme/templates/components/news-list/news-list.ui_patterns.yml

```diff
@@ -22,6 +22,11 @@ sdss-news-list:
       label: "Dek"
       description: "This is the dek specific to news items."
       preview: "<p>Sample text goes here and this list should be able to handle a few lines.</p>"
+    news_list_long_dek:
+      type: text
+      label: "Long Dek"
+      description: "This is the long dek specific to news items."
+      preview: "<p>Sample text goes here and this list should be able to handle a few lines.</p>"
```

Comment:
Do we need to add a new long dek field to the component? The Dek (long) field was just to allow them to use a longer dek in place of the current dek field, so we can just use the existing dek field in the component.

Reply:
I had thought we'd want to use the longer dek so it was less confusing to later devs.

Reply:
The name is fine. But if we are only ever going to display one dek field, the component should only have one dek field. Then the site builder chooses in the "Manage Display" tab for the component which field from the News content type (original dek field or dek (long) field) to use for the dek field display in the component. I hope that makes sense!

Reply:
I'm not exactly sure what you would like on this file.  Should I remove the long dek from the patterns.yml file?